featuros-plugin/src/main/java/io/fourfinanceit/featuros/Featuros.java

```diff
@@ -0,0 +1,66 @@
+package io.fourfinanceit.featuros;
+
+import jodd.http.HttpException;
+import jodd.http.HttpResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URI;
+import java.util.Optional;
+
+import static jodd.http.HttpRequest.post;
+
+/**
+ * Plugin that reaches out to featuros dashboard and notifies of current deployment.
+ * When not using a boot-starter module, it has to be called manually, e.g.:
+ * <pre>
+ *     public static void main(String... args) {
+ *         SpringApplication.run(args);
+ *         Featuros.notification();
+ *     }
+ * </pre>
+ */
+public class Featuros {
+
+    private static final Logger log = LoggerFactory.getLogger(Featuros.class);
```

Comment:
It's shorter but then you can easily copy paste the thing :) BTW I'd suggest that you add Lombock project then you'll just annotate the class with Slf4j or sth and won't have to add anything.